ignite/metrics/frequency.py

```diff
@@ -70,10 +66,11 @@ def compute(self):
         # Returns the average processed objects per second across all workers
         return self._n / self._elapsed * time_divisor
 
-    def completed(self, engine, name):
+    def completed(self, engine: Engine, name: str) -> None:
         engine.state.metrics[name] = int(self.compute())
 
-    def attach(self, engine, name, event_name=Events.ITERATION_COMPLETED):
+    # TODO: see issue https://github.com/pytorch/ignite/issues/1405
+    def attach(self, engine: Engine, name: str, usage: Union[str, MetricUsage] = EpochWise()) -> None:
```

Comment:
I was thinking to keep the signature as it was : `attach(self, engine, name, event_name=Events.ITERATION_COMPLETED)` and an ignore mark for mypy.
Otherwise, `engine.add_event_handler(usage, self.completed, name)` looks wrong.

Reply:
ok, gotcha 👍